tests/test_relation.py

```diff
@@ -36,6 +36,12 @@ def setup_class(cls):
         cls.img = schema.Image()
         cls.trash = schema.UberTrash()
 
+    def test_class_help(self):
+        help(schema.TTest)
```

Comment:
would it make sense to also e.g.:
`assert schema.TTest.definition in schema.TTest.__doc__`

Reply:
done. Also moved into `test_declare`